Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -9,17 +9,17 @@ Basics
 ======
 
 .. note::
-   Site handling as described here is available since TYPO3 9 LTS.
+   Site handling is available since TYPO3 9 LTS.
 
-TYPO3 site handling and configuration is the starting point for creating new web sites. The corresponding modules are found in the TYPO3 backend
-in the section "Site management".
+TYPO3 site handling and configuration is the starting point for creating new web sites. 
+The corresponding modules are found in the TYPO3 backend in the section "Site management".
 
 A site configuration consists of the following parts:
 
-* Base URL configuration: Under which domain(s) is my site accessible
-* :ref:`Language configuration<sitehandling-addingLanguages>`: Which languages are available for my site
-* :ref:`Error Handling<sitehandling-errorHandling>`: How should errors for this site behave (For example: configure custom 404 pages)
-* :ref:`Static Routes<sitehandling-staticRoutes>`: Add static routes to a site (For example for robots.txt on a per site base)
+* Base URL configurations: Under these domain(s) my site is accessible.
+* :ref:`Language configuration<sitehandling-addingLanguages>`: These languages are available for my site.
+* :ref:`Error Handling<sitehandling-errorHandling>`: How should errors for this site behave (For example: configuration of custom 404 pages)
```

Comment:
```suggestion
* :ref:`Error Handling<sitehandling-errorHandling>`: error behavior of my site (For example: configuration of custom 404 pages)
```

Reply:
Which English is to be used for TYPO3?

Reply:
A good question but right now there is no strict rule here. Personally I'd try to aim at American English though.